governance-exercises/JanSky Polling Exercise 2 - Potential Aligned Delegate Privacy Breach - July 31, 2023.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Potential Aligned Delegate Privacy Breach - July 31, 2023
+summary: Signal your support or opposition to declaring AD Clive in breach of the privacy standards of the Atlas (section 2.6.6). 
+discussion_link: link_to_my_announcement_thread 
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-07-31T16:00:00
+end_date: 2023-08-03T16:00:00
+---
+# Poll: Potential Aligned Delegate Privacy Breach - July 31, 2023
+
+The Governance Facilitator has placed a Governance Poll into the voting system on behalf of the Maker Community. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, July 31 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following statements:
+* Based on the evidence submitted by Ecosystem Diane and other available sources, Aligned Delegate Clive is in breach of [Section 2.6.6: Aligned Delegate Privacy](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy) of the Atlas.
+* AD Clive should be derecognized from his position, and stripped of his Alignment Conserver status. 
```

Comment:
Expressing this as agreement to a 'should' statement is probably not great. There are more neutral ways to present this in a poll.